src/bootstrap/bootstrap.py

```diff
@@ -1121,15 +1121,15 @@ def check_vendored_status(self):
         if self.use_vendored_sources:
             if not os.path.exists('.cargo'):
                 os.makedirs('.cargo')
-            with output('.cargo/config') as cargo_config:
-                cargo_config.write(
-                    "[source.crates-io]\n"
-                    "replace-with = 'vendored-sources'\n"
-                    "registry = 'https://example.com'\n"
-                    "\n"
-                    "[source.vendored-sources]\n"
-                    "directory = '{}/vendor'\n"
-                    .format(self.rust_root))
+                with output('.cargo/config') as cargo_config:
+                    cargo_config.write(
+                        "[source.crates-io]\n"
+                        "replace-with = 'vendored-sources'\n"
+                        "registry = 'https://example.com'\n"
+                        "\n"
+                        "[source.vendored-sources]\n"
+                        "directory = '{}/vendor'\n"
+                        .format(self.rust_root))
```

Comment:
I think we should still dump this to stdout or otherwise tell the user they probably want to add it (or otherwise configure it).

Reply:
I updated the PR, adding some "info" about this. Was what you mean?